src/main/java/org/jabref/logic/util/FileType.java

```diff
@@ -1,105 +1,18 @@
-    INSPEC(Localization.lang("%0 file", "INSPEC"), "txt"),
-    ISI(Localization.lang("%0 file", "ISI"), "isi", "txt"),
-    ISO_690_RTF(Localization.lang("%0 file", "ISO 690"), "rtf"),
-    ISO_690_TXT(Localization.lang("%0 file", "ISO 690"), "txt"),
-    MEDLINE(Localization.lang("%0 file", "Medline"), "nbib", "xml"),
-    MEDLINE_PLAIN(Localization.lang("%0 file", "MedlinePlain"), "nbib", "txt"),
-    MIS_QUARTERLY(Localization.lang("%0 file", "MIS Quarterly"), "rtf"),
-    MODS(Localization.lang("%0 file", "MODS"), "xml"),
-    MSBIB(Localization.lang("%0 file", "MS Office 2007 bib"), "xml"),
-    OO_LO(Localization.lang("%0 file", "OpenOffice/LibreOffice"), "csv"),
-    OVID(Localization.lang("%0 file", "Ovid"), "txt"),
-    PDF_CONTENT(Localization.lang("%0 file", "PDF content "), "pdf"),
-    PUBMED(Localization.lang("%0 file", "PubMed"), "fcgi"),
-    REPEC("REPEC New Economic Papers (NEP)", "txt"),
-    RIS(Localization.lang("%0 file", "RIS"), "ris"),
-    SILVER_PLATTER(Localization.lang("%0 file", "SilverPlatter"), "dat", "txt"),
-    SIMPLE_HTML(Localization.lang("%0 file", Localization.lang("Simple HTML")), "html"),
-    PDF_XMP(Localization.lang("XMP-annotated PDF"), "pdf"),
-    PLAIN_XMP(Localization.lang("%0 file", "XMP"), "xmp"),
-
-    AUX(Localization.lang("%0 file", "AUX"), "aux"),
-    JSTYLE(Localization.lang("Style file"), "jstyle"),
-    LAYOUT(Localization.lang("Custom layout file"), "layout"),
-    TERMS(Localization.lang("Protected terms file"), "terms"),
-    TXT(Localization.lang("%0 file", Localization.lang("Plain text")), "txt"),
-    CLASS(Localization.lang("%0 file", "CLASS"), "class"),
-    JAR(Localization.lang("%0 file", "JAR"), "jar"),
-    XML(Localization.lang("%0 file", "XML"), "xml"),
-    ZIP(Localization.lang("%0 file", "ZIP"), "zip"),
-    ODS(Localization.lang("%0 file", Localization.lang("OpenDocument spreadsheet")), "ods"),
-    SXC(Localization.lang("%0 file", "OpenOffice/LibreOffice Calc"), "sxc"),
-    HTML(Localization.lang("%0 file", "HTML"), "html"),
-    RTF(Localization.lang("%0 file", "RTF"), "rtf"),
-    RDF(Localization.lang("%0 file", "RDF"), "rdf"),
-    CSV(Localization.lang("%0 file", "CSV"), "csv"),
-    DEFAULT(Localization.lang("%0 file", "DEFAULT"), "default");
-
-    private static final EnumSet<FileType> ALL_FILE_TYPES = EnumSet.allOf(FileType.class);
-    private final String[] extensions;
-    private final String description;
-
-    FileType(String description, String... extensions) {
-        this.description = description;
-        this.extensions = extensions;
-    }
-
-    public List<String> getExtensions() {
-        return Arrays.asList(extensions);
-    }
+public interface FileType {
 
-    public String getDescription() {
-        StringJoiner sj = new StringJoiner(", ", description + " (", ")");
-        for (String ext : extensions) {
-            sj.add("*." + ext);
-        }
-        return sj.toString();
-    }
+    String getDescription();
```

Comment:
Also on a second look, I'm strongly in favor of removing the `description` from the file type. It is actually the description of the exporter/importer that we should show in the dialog...if you want to reuse the strings you can extract the descriptions in `BasicFileType` to a new class, but its also ok if you just repeat the description in the importer and exporter.

Reply:
This would not solve the underlying problem and in fact would result in the same problem I intend to fix here. 
As we have multiple file types with the same extension e.g. html. We already have Html Table, Html LIst etc. Only with the added description, the file type is unique

Reply:
I think your example is a good illustration why the current mix of file extensions and description is misleading. The file type is `html` in both examples, but the exporter used is different (tables vs list). You don't get these kind of problems if the `FileType` is only an abstraction of the extension and the description comes from the exporter...

Reply:
At least we need the description in the extension filter. Otherwise we can't map it. I will think about this. I will look and see how this would affect the importer